tests/Agent/IntegrationTests/IntegrationTests/AgentFeatures/BrowserAgentAutoInjection6Plus.cs

```diff
@@ -0,0 +1,87 @@
+// Copyright 2020 New Relic, Inc. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using NewRelic.Agent.IntegrationTestHelpers;
+using NewRelic.Agent.IntegrationTests.RemoteServiceFixtures;
+using NewRelic.Testing.Assertions;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace NewRelic.Agent.IntegrationTests.AgentFeatures
+{
+    public class BrowserAgentAutoInjection6PlusBase : NewRelicIntegrationTest<BasicAspNetCoreRazorApplicationFixture>
+    {
+        private readonly BasicAspNetCoreRazorApplicationFixture _fixture;
+        private string _htmlContent;
+        private string _staticContent;
+
+        public BrowserAgentAutoInjection6PlusBase(BasicAspNetCoreRazorApplicationFixture fixture,
+            ITestOutputHelper output, bool enableResponseCompression)
+            : base(fixture)
+        {
+            _fixture = fixture;
+            _fixture.TestLogger = output;
+            _fixture.Actions
+            (
+                setupConfiguration: () =>
+                {
+                    var configPath = fixture.DestinationNewRelicConfigFilePath;
+
+                    var configModifier = new NewRelicConfigModifier(configPath);
+                    configModifier.AutoInstrumentBrowserMonitoring(true);
+                    configModifier.BrowserMonitoringEnableAttributes(true);
+
+                    configModifier.BrowserMonitoringLoader("rum");
```

Comment:
Should we add tests for the different browser agents that are available? For the Asp.Net browser tests, we are testing that the different browser agents work.

Reply:
I don't think so - the logic for loading a specific browser agent is unchanged for the ASP.NET Core 6+ browser injection.

Reply:
We received requests from the browser team a couple of times where they asked us to confirm if a certain type of browser agent was causing problems. We have test coverage for different string formats, and we somewhat recently added coverage for the 3 browser agent types in an aspnet app (which was the only thing we supported). It might save us some time if we have a similar set of tests for aspnetcore.

Reply:
I can add some tests, but I don't think they'll do anything useful - injecting the script is exactly the same, regardless of which browser agent type gets loaded.

Reply:
Right, I see this as more of an end to end test that has been requested from time to time, and it automates what we have been asked to do manually.